README.md

```diff
@@ -0,0 +1,198 @@
+
+#### Cloud Pub/Sub
+
+After enabling Pub/Sub, go to https://console.cloud.google.com/cloudpubsub?project=YOUR-PROJECT-ID . Note that you have to change *YOUR-PROJECT-ID* to, you guessed it, your project id.
+
+There, click "Create Topic" and type in whatever name you want to give to the topic we'll be using to read the events from. In our example, we used "test-topic". Keep track of your topic's name (you can always go to that URL to check it), as it will be needed when you're creating your config file.
+
+![creating a pubsub topic][create-topic]
+
+#### Cloud Storage
+Create a staging location in which the system will store the jars ran by the workers. To do this, you need to enable the Storage API.
+
+After that, go to https://console.cloud.google.com/storage/browser?project=YOUR-PROJECT-ID . Click "Create Bucket". Fill in the appropriate details. As this is an example project, we suggest you pick the cheapest option.
+
+After creating your bucket, you might want to create a folder inside it, to serve as your staging location. You can also just use the bucket's root. To create a folder in your bucket, after you created the bucket, select it in the list and then click "Create Folder".
+
+Your staging location will be:
+
+```bash
+gs://your-bucket-name/the-folder-that-you-might-have-created
+```
+
+![setting staging location][bucket1]
+
+![setting staging location][bucket2]
+
+#### Bigtable
+
+After enabling Bigtable, we'll need to create an instance where our table will be stored. And then create the actual table.
+
+##### Creating an instance
+
+Go to https://console.cloud.google.com/bigtable/instances?project=YOUR-PROJECT-ID , again taking care of changing it with your project id. Click "Create instance" and fill in the appropriate details. The name is, again, important. We used "test-instance" in our example. We suggest you to select HDD as storage type as it is much cheaper, and this is for testing purposes only.
+                  
+|                                            |                                            |
+|:------------------------------------------:|:------------------------------------------:|
+| ![creating a bigtable instance][bigtable1] | ![creating a bigtable instance][bigtable2] |
+
+##### Creating a table
+
+We usually do this through an HBase shell inside Google Cloud Shell ([reference](https://cloud.google.com/bigtable/docs/quickstart-hbase)).  
+
+Go to your Google Cloud Dashboard (https://console.cloud.google.com/home/dashboard?project=YOUR-PROJECT-ID) and click on the little terminal symbol on the top right corner. That will boot your Google Shell. Wait a bit and then run the following commands:
+
+```bash
```

Comment:
It duplicates commands present in above link (and could obsolete at some point). I'd remove this four commands or link to docs, or even use `inv create-table` way.